inc/saiport.h

```diff
@@ -572,6 +587,15 @@ typedef enum _sai_port_attr_t
     SAI_PORT_ATTR_INTERNAL_LOOPBACK_MODE,
 
     /**
+     * @brief Forward Error Correction (FEC) control
+     *
+     * @type sai_port_fec_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_FEC_MODE_NONE
```

Comment:
Are you thinking in terms of advertising FEC abilities with auto-negotiation?  If I go with a list, and someone configures in static mode and (mis-)configures both FEC options (like in 25G), which type takes precedence?